main.nf

```diff
@@ -68,6 +68,22 @@ params.vep_version           = WorkflowMain.getGenomeAttribute(params, 'vep_vers
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
+include { validateParameters; paramsHelp } from 'plugin/nf-validation'
+
+// Print help message if needed
+if (params.help) {
+    def logo = NfcoreTemplate.logo(workflow, params.monochrome_logs)
+    def citation = '\n' + WorkflowMain.citation(workflow) + '\n'
+    def String command = "nextflow run ${workflow.manifest.name} --input samplesheet.csv --genome GRCh38 -profile docker"
```

Comment:
```suggestion
    def String command = "nextflow run ${workflow.manifest.name} --input samplesheet.csv --genome GATK.GRCh38 -profile docker"
```

Reply:
shouldn't the default command contain --outdir as well?

Reply:
It's not there [now](https://github.com/nf-core/sarek/blob/master/lib/WorkflowMain.groovy#L27C71-L27C82), but it's true we should add it as it is required 👍